frontend/packages/core/src/NPS/emojiRatings.tsx

```diff
@@ -27,11 +32,18 @@ const getKey = (map, val): string => Object.keys(map).find(key => map[key] === v
  * @param setRating function which will return the given selected rating
  * @returns rendered EmojiRatings component
  */
-const EmojiRatings = ({ ratings = [], setRating }: EmojiRatingsProps) => {
+const EmojiRatings = ({
+  ratings = [],
+  setRating,
+  placement = "top",
+  buttonSize = "small",
+  emojiSize = "medium",
```

Comment:
Why do we have both button and emoji size?

Reply:
For extendability, and since they're no longer mirroring each other I wanted to separate them into their own props so we could override them as necessary.

Reply:
:+1: , just thinking about it though would we ever want to allow these to be that different stylistically?